
From: Hideo AOKI <aoki@sdl.hitachi.co.jp>

Attached patch is a short description for /proc/sys/vm/swap_token_timeout.

Signed-off-by: Hideo Aoki <aoki@sdl.hitachi.co.jp>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/filesystems/proc.txt |    8 ++++++++
 25-akpm/Documentation/sysctl/vm.txt        |    2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

diff -puN Documentation/filesystems/proc.txt~vm-thrashing-control-tuning-docs Documentation/filesystems/proc.txt
--- 25/Documentation/filesystems/proc.txt~vm-thrashing-control-tuning-docs	Thu Oct  7 13:59:19 2004
+++ 25-akpm/Documentation/filesystems/proc.txt	Thu Oct  7 13:59:19 2004
@@ -1295,6 +1295,14 @@ block_dump
 block_dump enables block I/O debugging when set to a nonzero value. More
 information on block I/O debugging is in Documentation/laptop-mode.txt.
 
+swap_token_timeout
+------------------
+
+This file contains valid hold time of swap out protection token. The Linux
+VM has token based thrashing control mechanism and uses the token to prevent
+unnecessary page faults in thrashing situation. The unit of the value is
+second. The value would be useful to tune thrashing behavior.
+
 2.5 /proc/sys/dev - Device specific parameters
 ----------------------------------------------
 
diff -puN Documentation/sysctl/vm.txt~vm-thrashing-control-tuning-docs Documentation/sysctl/vm.txt
--- 25/Documentation/sysctl/vm.txt~vm-thrashing-control-tuning-docs	Thu Oct  7 13:59:19 2004
+++ 25-akpm/Documentation/sysctl/vm.txt	Thu Oct  7 13:59:19 2004
@@ -31,7 +31,7 @@ Currently, these files are in /proc/sys/
 
 dirty_ratio, dirty_background_ratio, dirty_expire_centisecs,
 dirty_writeback_centisecs, vfs_cache_pressure, laptop_mode,
-block_dump:
+block_dump, swap_token_timeout:
 
 See Documentation/filesystems/proc.txt
 
_
